src/routes/store.js

```diff
@@ -302,6 +302,42 @@ function init (server, { middlewares, helpers } = {}) {
 
     return result
   }))
+
+  server.post({
+    name: 'store.syncCache',
+    path: '/store/platforms/:id/cache/sync'
+  }, allowSystem, wrapAction(async (req, res) => {
+    const {
+      id: platformId
+    } = req.params
+    const env = req.env
+
+    if (!env) throw createError(400, 'Missing environment')
+
+    return requester.send({
+      type: 'syncCache',
+      platformId,
+      env
+    })
+  }))
+
+  server.post({
```

Comment:
Why not use `DELETE`?

Do you have some parameters in mind for the future?

Reply:
Right, I use `POST` for this endpoint because the other drop endpoints also used the verb POST `/store/platforms/:id/elasticsearch/drop` and `/store/platforms/:id/database/drop`.

But this time, it's explicitly delete and there will have no parameters in the future.
Should we deprecate drop endpoints and use DELETE verb too?

Reply:
Yes, preferably in another PR.